src/js/typeahead/helpers.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 module.exports = {
-  sorter: function(input) {
-    return function sorter(a, b) {
+  sorter: function sorter(input) {
+    return function sort(a, b) {
```

Comment:
well obviously it wouldn't be just for the change above. It would mean our JS code would be consistent with itself, which seems like a plus to me. Babelify can just be dropped in